server/src/main/java/ru/hse/server/filter/JWTFilter.java

```diff
@@ -22,7 +22,7 @@
 public class JWTFilter extends OncePerRequestFilter {
     private static final String AUTHORIZATION_HEADER = "Authorization";
 
-    private final JWTProviderService jwtProviderService;
+    private final JWTProvider jwtProviderService;
```

Comment:
How this service initializes?

Reply:
With annotation RequiredArgsConstructor, it create constructor that expect JWTProvider